### PR TITLE
🐛 Designer Review - Media width problems

### LINKED ISSUE #113: 🐛 Designer Review - Media width problems
- [ ] Map disappears too early when enlarging screen + components aren't left aligned + YouTube video doesn't disappear

    Image
**Figure: New site with incorrect positioning and map gone**

    Image
**Figure: Current site with correct positioning and map still showing**

- [ ] When zoomed in far enough on the current site the "Upcoming Events" widget spans the full screen width and replaces the "Australia's Leading .NET and Azure Consultants" information. This does not happen on the new site.

    Image
**Figure: New site**

    Image
**Figure: Current site**

### PR DESCRIPTION
**Pull request by jimmidier**
Closes #113 

image
**Figure: 2 columns on sm breakpoint**

image
**Figure: Events widget now expand to full screen width on sm breakpoint**

image
**Figure: Images on column layout now have appropriate widths under md breakpoint**

### CODE REVIEW

File: components/blocks/aboutUs.tsx [hunk 1]
```diff
@@ -137,24 +141,24 @@ const TV = ({ className }) => {
...
-      {videoClicked ? (
-        <iframe
-          src={layoutData.aboutUs.video.url}
-          width="100%"
-          allowFullScreen
-        ></iframe>
-      ) : (
-        <>
-          {/* TODO: refactor with next/image */}
+      <div className="relative w-full h-0 pb-9/16">
```
Comment (wicksipedia) [inline components/blocks/aboutUs.tsx:144]: linting - tailwind class name order

**Review by wicksipedia — CHANGES_REQUESTED** [review]
TIP: select a device when using the responsive layout

The about us doesn't look right when compared to the webforms site (<https://www.ssw.com.au/ssw>):

1. all elements should be visible on tablet/desktop
2. video should **not** be visible on phone
3. map should **not** be visible on phone (portrait mode)

image
image

File: components/blocks/aboutUs.tsx (hunk 1 above)
Comment (jimmidier) [reply components/blocks/aboutUs.tsx]: Done - fixed

**Comment by jimmidier** [comment]
> The about us doesn't look right when compared to the webforms site (https://www.ssw.com.au/ssw):
> 
> 1. all elements should be visible on tablet/desktop
> 2. video should **not** be visible on phone
> 3. map should **not** be visible on phone (portrait mode)
> 

  > 1. all elements should be visible on tablet/desktop
Not sure if by "should be visible" you meant "align with the UI on the webforms site" or "make them visible regardlessly", as the tv video is hidden on tablet on the old site. Other breakpoints are now aligned with the old site.
image
**Figure: TV video is hidden on tablet**

  > 2 and 3
All done

**Review by wicksipedia — APPROVED** [review]

**Pull request merged**